test/e2e/framework/pod/utils.go

```diff
@@ -338,3 +342,21 @@ func IsPodOnCgroupv2Node(f *framework.Framework, pod *v1.Pod) bool {
 	}
 	return len(out) != 0
 }
+
+// TODO: Remove the rounded cpu limit values when https://github.com/opencontainers/runc/issues/4622
+// is fixed.
+func GetCPULimitCgroupExpectations(milliCPULimit int64) []string {
+	cpuQuota := kubecm.MilliCPUToQuota(milliCPULimit, kubecm.QuotaPeriod)
+	expectedCPULimit := strconv.FormatInt(cpuQuota, 10)
+	expectedCPULimits := []string{fmt.Sprintf("%s %s", expectedCPULimit, CPUPeriod)}
```

Comment:
I think we lost the `if *podOnCgroupv2Node {` condition around this formatting. We need this in the rounded case too. Maybe extract a helper?

Reply:
🤦‍♀️ sorry, thanks for catching that. updated